umbraco-cloud/set-up/project-settings/umbraco-cicd/Troubleshooting.md

```diff
@@ -11,6 +11,28 @@ To resolve this issue, remove the `RestorePackagesWithLockFile` to allow the dep
 
 ## cloud-sync
 
+### The projects left-most environment has changed
+The mechanism to determine changes since last deployment is not able to do so when the left-most environment has changed. This happens when you either add or remove the development environment.
```

Comment:
```suggestion
The mechanism to determine changes since the last deployment is not able to do so when the left-most environment has changed. This happens when you either add or remove the development environment.
```